src/System.Memory/src/System/SpanHelpers.byte.cs

```diff
@@ -80,16 +77,13 @@ public static int LastIndexOfAny(ref byte searchSpace, int searchSpaceLength, re
             if (valueLength == 0)
                 return 0;  // A zero-length sequence is always treated as "found" at the start of the search space.
 
-            int index = -1;
+            int index = int.MinValue;
```

Comment:
You may be able to just use `int index = -1;` here, and then you do not need to normalize it at the end.

Reply:
Yeah, that makes sense on LastIndexOf -- it appears I didn't think too much about it, only about the "forward-version".

Reply:
I'll apply the same in SpanHelper.T.cs LastIndexOf-method.